examples/benchmark/tls_bench.c

```diff
@@ -59,8 +61,7 @@ bench_tls();
 
 /* configuration parameters */
 #define THREAD_COUNT   2
```

Comment:
Can you also add an argument for thread pairs `-t` and default to 1? Currently it defaults to 2, which actually starts 2 clients and 2 servers and combines their throughput. I think it would be more clear to have 1 pair of threads by default.

Reply:
'-t' is already being used for time. I'll make it '-T'.